docs/development/design/organizations.rst

```diff
@@ -0,0 +1,145 @@
+Organizations
+=============
+
+Currently we don't support organizations in the community site
+(a way to group different projects),
+we only support individual accounts.
+
+Several integrations that we support have organizations,
+where users group their repositories.
+
+Why move organizations in the community site?
+---------------------------------------------
+
+We support organizations in the commercial site,
+having no organizations in the community site makes the code maintenance difficult.
+Having organizations in the community site will make the differences between both more easy to manage.
+
+Users from the community site can have organizations in external sites from where we import their projects
+(like GitHub, Gitlab).
+Currently users have all projects from different organizations in their account.
+Having not a clear way to group/separate those.
+
+There is also a product decision.
+Supporting organizations only on code,
+but enable the feature only on the commercial site.
+
+How are we going to support organizations?
+------------------------------------------
+
+Currently only users can own projects.
+With organizations this is going to change to: 
+Users and organizations can own projects.
+
+With this, the migration process would be straightforward for the community site.
+
+For the commercial site we still have a decision.
+Are we going to support users and organizations to own projects?
+Or just organizations?
```

Comment:
I need to think a little more about this. At first sight, it will bring some issues with the unique slugs for the commercial site. If a user owns a project in commercial, what would be the slug we will generate? `username` + `project.slug`?

Reply:
Well, we have to support this on the community site with only `project.slug`, so for the commercial site it would be the same, only `project.slug`.

Reply:
But on the commercial site the slug doesn't matter that much anyway, users use a custom domain instead.

Reply:
I don't think it's a good idea to allow just `project.slug` in the corporate site for slugs.

Basic plan does not have a Custom Domain.

Making collisions on Commercial will only confuse users, IMO.

Reply:
Users and Orgs can both own projects. It's the only reasonable way to transition it I think, unless we hit some kind of serious issue with the code that won't support it.

Reply:
On the commercial site, i feel like it should *only* be organizations, as there is no per-user subscription. Can we make this a setting -- even if we just use the setting to alter the list of target relations for the project? On commercial, we'd limit this to organzations the user has perimssions to import into, on community the list would be a list of user/organizations the user can import into.

Reply:
> as there is no per-user subscription

That's a good point. Yeah, I think we can manage that with a setting.